### DISCUSSION TITLE
10 GB used after "complete cleanup"

### DISCUSSION THREAD

**Discussion by ghost** [AbdBarho/stable-diffusion-webui-docker#593] [Q&A]
I have been playing around with this repo and docker following the instructions. 

Before I started I had 73.3 GB used on my disk. When running it went up to 106 GB. Afterwards I deleted the container, all images and the cloned repo. However the hdd usage went down just to 83.3 GB. 

I have tried running docker image prune and container prune but there is nothing to remove. 

Anyone has an Idea, what could have happened? How do I free up again these 10 GB? 

Thanks in advance!

**Comment by DevilaN** [comment]
You started without docker installed or with it?

**Reply by ghost** [reply]
Good point. Now that you said that, I remember checking the installation of docker and I saw I had some old packages installed according to: 
 https://docs.docker.com/engine/install/ubuntu/ 

I uninstalled them as indicated and installed the new ones.  

I will check how much disk space they need to be sure. 

Than you very much!

**Reply by AbdBarho** [reply]
it could also be the docker build cache, which is not pruned when you clear the images or the containers

https://docs.docker.com/engine/reference/commandline/builder_prune/

**Reply by ghost** [reply]
That was exactly it!!! Thank you very much! 
![Screenshot from 2023-10-07 19-11-33](https://github.com/AbdBarho/stable-diffusion-webui-docker/assets/53519166/df2a85e9-fdac-4361-8c9f-d7675e6393dc)
